bootstrap/command.go

```diff
@@ -148,7 +148,7 @@ func Main(ctx *blueprint.Context, config interface{}, extraNinjaFileDeps ...stri
 	deps = append(deps, extraDeps...)
```

Comment:
I suspect a dependency somewhere, possibly this <deps> , needs to include the $BLUEPRINT_LIST_FILE in order to rerun the right invocations of Blueprint when a new Android.bp is created.

Reply:
Yeah that ended up being added in the next patchset